lib/bootloader_setup.pm

```diff
@@ -88,14 +88,16 @@ sub bootmenu_default_params {
     if (check_var('VIRSH_VMM_FAMILY', 'hyperv')) {
         type_string " video=hyperv_fb:1024x768";
     }
+    return if is_jeos || is_casp('VMX');
```

Comment:
There is no yast on qcow images, they are set up with cloud-init. Y2DEBUG is still set for ISO image (would be is_casp('DVD'))